10 - ARM/S_ARM_02 RGB LED Control V1.0/Project - RGB LED Control V1.0/app_interface.h

```diff
@@ -0,0 +1,47 @@
+/*
+ * app_interface.h
+ *
+ *   Created on: Jun 10, 2023
+ *       Author: Abdelrhman Walaa - https://github.com/AbdelrhmanWalaa
+ *  Description: This file contains all Application (APP) functions' prototypes and definitions (Macros) to avoid magic numbers.
+ */ 
+
+#ifndef APP_INTERFACE_H_
+#define APP_INTERFACE_H_
+
+/*******************************************************************************************************************************************************************/
+/* APP Includes */
+
+/* LIB */
+#include "std_types.h"
+#include "bit_math.h"
+
+/* HAL */
+#include "led_interface.h"
+#include "btn_interface.h"
+
+/* MCAL */
+#include "delay_interface.h"

```

Comment:
You shouldn't include drivers from MCAL layer in application

Reply:
Thanks for your feedback🤲🏼, however I justified this in the design document.